tests/calc/test_indices.py

```diff
@@ -176,6 +176,51 @@ def test_bunkers_motion():
     assert_almost_equal(motion.flatten(), truth, 8)
 
 
+def test_corfidi_motion():
```

Comment:
This is fundamentally 5 separate tests. Can you separate into 5 individual, appropriately named test functions? This makes it easier to understand what cases are being tested. Also, as currently structured, the test stops running when the first assert fails, so refactoring ensures all cases have a chance to run.

Reply:
makes sense! I'm fairly new to the concept of automated testing so this is good to consider.

Reply:
Absolutely! I'll add another positive of breaking up is that you're less likely to have accidental coupling between the tests (i.e. a later test relying on something implicitly done above).